infra/charts/feast/charts/feast-jobservice/values.yaml

```diff
@@ -0,0 +1,154 @@
+# replicaCount -- Number of pods that will be created
+replicaCount: 1
+
+image:
+  # image.repository -- Docker image repository
+  repository: gcr.io/kf-feast/feast-jobservice
+  # image.tag -- Image tag
+  tag: develop
+  # image.pullPolicy -- Image pull policy
+  pullPolicy: IfNotPresent
+
+application.yaml:
+  # "application.yaml".enabled -- Flag to include the default [configuration](https://github.com/feast-dev/feast/blob/master/job-controller/src/main/resources/application.yml). Please set `application-override.yaml` to override this configuration.
+  enabled: true
+
+application-generated.yaml:
+  # "application-generated.yaml".enabled -- Flag to include Helm generated configuration for http port, Feast database URL, Kafka bootstrap servers and jobs metrics host. This is useful for deployment that uses default configuration for Kafka, Postgres and StatsD exporter. Please set `application-override.yaml` to override this configuration.
+  enabled: true
+
+# "application-secret.yaml" -- Configuration to override the default [application.yaml](https://github.com/feast-dev/feast/blob/master/job-controller/src/main/resources/application.yml). Will be created as a Secret. `application-override.yaml` has a higher precedence than `application-secret.yaml`. It is recommended to either set `application-override.yaml` or `application-secret.yaml` only to simplify config management.
+application-secret.yaml:
+  enabled: true
+
+# "application-override.yaml" -- Configuration to override the default [application.yaml](https://github.com/feast-dev/feast/blob/master/job-controller/src/main/resources/application.yml). Will be created as a ConfigMap. `application-override.yaml` has a higher precedence than `application-secret.yaml`
+application-override.yaml:
+  enabled: true
```

Comment:
^ application yaml stuff is springboot specific so we won't use it with a python service